packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -62,10 +63,14 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme) => {
   };
 });
 
-export const Tab: FC<TabProps> = ({ label, active, icon, onChangeTab }) => {
+export const Tab: FC<TabProps> = ({ label, active, icon, onChangeTab, hideFromTabs }) => {
   const theme = useTheme();
   const tabsStyles = getTabStyles(theme);
 
+  if (hideFromTabs) {
```

Comment:
Do we really need this? I think we should simply NOT render a tab, if it's not necessary.

In place where you would possibly need it:
```
{ !hideFromTabs && <Tab ... /> }
```